package-lock.json

```diff
@@ -58349,7 +58326,7 @@
                 "express": "^4.16.3",
                 "filesize": "^3.6.1",
                 "gzip-size": "^5.0.0",
-                "lodash": "^4.17.15",
+                "lodash": ">=4.17.21",
```

Comment:
I don't think these are necessary? Doesn't ```^4.17.15``` mean that it will accept any ```4``` version? So your change to the package.json should cause the rest of these dependencies to just use the version we ship.

Reply:
Meaning you don't need the hand editing here?

Reply:
Done.